ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -448,9 +448,16 @@ class Measure extends Annotation {
     }
 
     public addControlPoints(): void {
+        const options = this.options.controlPointOptions;
         let selectType = this.options.typeOptions.selectType,
             controlPoint;
 
+        if (selectType === 'x') {
+            options.style.cursor = 'ew-resize';
+        } else if (selectType === 'y') {
+            options.style.cursor = 'ns-resize';
+        }
```

Comment:
I think this should be changed only when a developer has not defined `style.cursor`